src/main/python/server.py

```diff
@@ -580,4 +758,4 @@ def my_form():
 
 
 if __name__ == '__main__':
-    app.run()
+    app.run()
```

Comment:
Not mandatory for this PR but more like maintenance work that is pending: This file has become too big. It should be modularized into functions, separated by functionality (I/O, drawing, ...) into different files and tests should be added for the functions.